Assets/Scripts/Gamestate/PodiumController.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PodiumController : MonoBehaviour
+{
+    [SerializeField]
+    private AugmentType augmentType;
+
+    private List<PlayerManager> playersInRadius;
+
+    private MatchController matchController = MatchController.Singleton;
+
+    void Start()
+    {
+    
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        PlayerManager player = GetComponentInChildren<PlayerManager>();
```

Comment:
**??????**

How does this PodiumController have the specific player it collides with as a child?

Reply:
Podium is kill